content/id/docs/concepts/storage/dynamic-provisioning.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Penyediaan Volume Dinamis
+content_template: templates/concept
+weight: 40
+---
+
+{{% capture overview %}}
+
+Penyediaan volume dinamis memungkinkan volume penyimpanan untuk dibuat sesuai permintaan (_on-demand_).
+Tanpa adanya penyediaan dinamis (_dynamic provisioning_), untuk membuat volume penyimpanan baru, admin kluster haruslah secara manual
+memanggil penyedia layanan cloud atau layanan penyimpanan, dan kemudian membuat [objek PersistentVolume](/docs/concepts/storage/persistent-volumes/)
+sebagai representasi di Kubernetes. Fitur penyediaan dinamis menghilangkan kebutuhan admin kluster untuk menyediakan
+penyimpanan sebelumnya (_pre-provision_). Namun, penyimpanan akan tersedia secara otomatis
+ketika diminta oleh pengguna.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Latar Belakang
+
+Penyediaan volume dinamis diimplementasi berdasarkan objek API StorageClass dari
+grup API `storage.k8s.io`. Seorang admin kluster dapat mendefinisikan berbagai macam
+objek StorageClass sesuai kebutuhan, masing-masing menentukan *plugin volume* (disebut
+juga *provisioner*) yang menyediakan sebuah volume beserta kumpulan parameter untuk
+diteruskan oleh _provisioner_ ketika proses penyediaan.
+
+Seorang kluster admin dapat mendefinisikan dan mengekspos berbagai templat penyimpanan
+(dari sistem penyimpanan yang sama maupun berbeda) di dalam kluster, masing-masing dengan
+kumpulan parameter tertentu. Desain ini memastikan bahwa pengguna tidak perlu merisaukan
+rumitnya bagaimana sebuah penyimpanan tersediakan, tapi tetap memiliki kemampuan untuk
```

Comment:
merisaukan rumitnya -> khawatir betapa rumitnya mekanisme penyediaan penyimpanan

Reply:
addressed in 490d25dd8a13b115293327a3e7ad9009052f2212